src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -391,6 +391,79 @@ std::vector<std::string> disabledTestPatterns() {
     retVector.emplace_back(R"(.*smoke_CompareWithRefs_static/EltwiseLayerTest.*_eltwise_op_type=Div_.*_model_type=i32_.*)");
 #endif
 
+#if defined(OPENVINO_ARCH_RISCV64)
+    // unsupported node type 'CausalMaskPreprocess'
+	retVector.emplace_back(R"(CausalMaskPreprocessCausalMaskPreprocess.smoke_CompareWithRefs)");
+    // fused op FakeQuantize has not been found
+	retVector.emplace_back(R"(ConvAndFQWithSharedConstants.smoke_ConvAndFQWithSharedConstants_CPU)");
+    // subgraphs code-generator is not supported on non-x64 platforms
+	retVector.emplace_back(R"(SubgraphSnippetSerializationTest.smoke_SerializeSubgraph(WithScalarConst)?)");
+	retVector.emplace_back(R"(.*SubgraphWithBlockedFormat.*)");
+    // unsupported layout
+	retVector.emplace_back(R"(GatherAddAvgpool.smoke_CompareWithRefs)");
+	retVector.emplace_back(R"(smoke_StaticAdaPoolAvg(4|5)DLayoutTest/AdaPoolLayerCPUTest.*_outFmts=nd?hwc_1)");
```

Comment:
Could you please replace tabs with 4 spaces? In the code of the our project we use 4 spaces for tabs

Reply:
Sure! I will replace all the tabs with 4 spaces! :)